Windows Executable - missing mavsdk.dll

Hi,
After having build issues several days earlier, I managed to successfully build and install MAVSDK. However, after creating and building a cmake C++ project and running the executable, I was given this error:
![image](https://user-images.githubusercontent.com/59535719/150616852-398aace3-fcdf-49b6-8179-9925940e6eac.png)
I think my CMakeLists.txt is fine but this is what it looks like:
![image](https://user-images.githubusercontent.com/59535719/150618265-99b23bac-4dbe-425b-9e2e-4920ef1bb374.png)
**Want to note that I did a system-wide install and I'm not sure if this should be done on Windows.
Also I noticed that the installed library found in C:\Program Files only contains mavsdk.lib and does not contain any .dll.**

I modified the CMakeLists.txt to include that line of code and was given this: ```CMake Error at src/mavsd/core/CMakeLists.txt:98 (message): CMAKE_INSTALL_LIBDIR: lib```

--- a/src/mavsdk/core/CMakeLists.txt
+++ b/src/mavsdk/core/CMakeLists.txt
@@ -91,6 +91,7 @@ target_include_directories(mavsdk
 
 install(TARGETS mavsdk
     EXPORT mavsdk-targets
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     )

